usr/src/uts/armv8/io/ns16550a/ns16550a.c

```diff
@@ -1969,18 +1969,18 @@ ns16550_baudok(struct ns16550com *ns16550)
 uint_t
 ns16550intr(caddr_t argns16550)
 {
-	struct ns16550com		*ns16550 = (struct ns16550com *)argns16550;
-	struct nsasyncline	*nsasync;
+	struct ns16550com	*ns16550 = (struct ns16550com *)argns16550;
+	struct nsasyncline	*nsasync = ns16550->ns16550_priv;
 	int			ret_status = DDI_INTR_UNCLAIMED;
-	uchar_t			interrupt_id, lsr;
+	uchar_t			lsr;
```

Comment:
Move this inside the `for` loop and make it `const` there like [in upstream](https://github.com/illumos/illumos-gate/commit/605f1b7c74747e9cf5add0f0a0b989bc832b8b20#diff-c298952be6c3d911ccc044b0a2491d72588d4d0601809de6b1722cc6063bcfb3R2283-R2284)?

Reply:
Sure, updated.

Reply:
Thanks.